api/types/mount/mount.go

```diff
@@ -23,9 +23,11 @@ type Mount struct {
 	// Source specifies the name of the mount. Depending on mount type, this
 	// may be a volume name or a host path, or even ignored.
 	// Source is not supported for tmpfs (must be an empty value)
-	Source   string `json:",omitempty"`
-	Target   string `json:",omitempty"`
-	ReadOnly bool   `json:",omitempty"`
+	Source       string `json:",omitempty"`
+	Target       string `json:",omitempty"`
+	ReadOnly     bool   `json:",omitempty"`
+	MaxBandwidth int    `json:",omitempty"`
+	MaxIOps      int    `json:",omitempty"`
```

Comment:
Should these be `uint` since it's minimum 0 ?

A minor issue, but could we make these `uint64` ? When we get this struct generated from the swagger spec it's going to generate them as `uint64`. It doesn't do `int` or `uint`, because that's technically ambiguous.